### DISCUSSION TITLE
How can I handle account change for eip155 !

### DISCUSSION THREAD

**Discussion by abhishekDabba** [reown-com/appkit#3245] [Q&A]
so I am using multichain sdk (ethers + solana )

I see the account change for eip155 is not reflecting on connect button even if I connect another account it still does not reflect new address 

any guess on how can I handle it ?

**Comment by anjumkaiser** [comment]
same issue for me,

is there any way to listen to accounts_change chains _change event like with @walletconnect/sign-client .
